script.js

```diff
@@ -0,0 +1,66 @@
+const formData = document.querySelector('.form');
+const data = document.querySelector('.data');
+
+function addNewBookData() {
+  const bookTitle = formData.title.value;
+  const bookAuthor = formData.author.value;
+
+  const book = {
+    title: bookTitle,
+    author: bookAuthor,
+  };
+
+  return book;
+}
+// store Books data in Local storage
+function storeBooks(book) {
+  let books = [];
+
+  if (localStorage.getItem('Books')) {
+    books = JSON.parse(localStorage.getItem('Books'));
+  }
+
+  books.push(book);
+  localStorage.setItem('Books', JSON.stringify(books));
+}
+// Adding Books data or retrieving data from local storage
+function addBook(element) {
+  element.preventDefault();
+  storeBooks(addNewBookData());
+  formData.submit();
+}
+// Displaying Books Data on the users Screen
+function displayBooks() {
+  if (localStorage.getItem('Books')) {
+    const books = JSON.parse(localStorage.getItem('Books'));
+    books.forEach((book) => {
+      const booksHtml = `
+        <div class="book-info">
+          <p class="title">${book.author}</p>
+          <p class="author">${book.title}</p>
+         <button class="remove-btn">Remove</button>
+         <hr>
+        </div>
+      `;
+      data.innerHTML += booksHtml;
+    });
+  }
+}
+displayBooks();
+// Target the submit button to add data to local storage.
+formData.addEventListener('submit', addBook);
+function removeBook(index) {
+  if (localStorage.getItem('Books')) {
+    const books = JSON.parse(localStorage.getItem('Books'));
+    books.splice(index, 1);
+    localStorage.clear();
+    localStorage.setItem('Books', JSON.stringify(books));
+  }
+}
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/211372819-971c3e26-8424-45a9-a8d9-387042960f60.png)

Please make sure that your remove function is removing the corresponding element 😉. Currently, there is a bug happening when the user removes multiple books:

If I have 5 books:
![image](https://user-images.githubusercontent.com/42799579/211373187-f07ba119-6d14-47c1-85d5-4c5dabcb4a85.png)

And remove books number 3 & 4:
![image](https://user-images.githubusercontent.com/42799579/211373304-ed974786-2398-4af8-a65d-3fe7866ded9e.png)

The localStorage isn't updating correctly:
![image](https://user-images.githubusercontent.com/42799579/211373352-10d1da07-ea8e-4ad8-b5b0-1e713dabb7ba.png)

After refreshing the page:
![image](https://user-images.githubusercontent.com/42799579/211373393-e8d8fd02-142f-414b-8a73-2a8c75d73eac.png)

This is happening because your variable index:
![image](https://user-images.githubusercontent.com/42799579/211373831-1507dc9b-c875-4aab-872c-b45cf7ce94de.png)
It isn't being updated after removing a book. As you can see, this is the index value when removing these books:
![image](https://user-images.githubusercontent.com/42799579/211374001-ae8aeb26-965e-4f49-952d-e185fa50270f.png)

The book number 3 index is correctly set as 2, while the book number 4 index was correctly set as 3. Although, after removing book number 3, the book number 4 index should be updated to 2, which is not happening. Because of this, the book that is being removed is book number 5, now indexed as 3. 

A good solution for this problem would be giving each book an id 😉. Cheers and keep coding!